models/points.go

```diff
@@ -55,6 +55,10 @@ var (
 
 	// ErrInvalidPoint is returned when a point cannot be parsed correctly.
 	ErrInvalidPoint = errors.New("point is invalid")
+
+	// ErrInvalidKevValuePairs is returned when the number of key, value pairs
+	// is odd, indicating a missing value.
+	ErrInvalidKevValuePairs = errors.New("key, value pairs is an odd length")
```

Comment:
This is a weird nit but can this be `key/value` instead of `key, value`? I feel like it could read weird depending on the context the error is displayed within.

Reply:
I will change the error text 👍 

As it happens, I was back and forth on this too. I chose this format because [`strings.NewReplacer`](https://golang.org/pkg/strings/#NewReplacer) is documented with the words _old, new string pairs_.

Reply:
Done